pkgs/games/purpur/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchurl, nixosTests, jre_headless, makeWrapper }:
+stdenv.mkDerivation rec {
```

Just a minor formatting issue:
```suggestion
{ lib, stdenv, fetchurl, nixosTests, jre_headless, makeWrapper }:

stdenv.mkDerivation rec {
```
btw, i'd recommand running `nixpkgs-fmt` on your derivations to do trivial stuff like this automatically.